README.md

```diff
@@ -90,14 +90,14 @@ $ az login
 $ az account set --name "<SUBSCRIPTION NAME OR ID>"
 
 $ az group create \
```

Comment:
I thought of making a PR but since you already have a PR for the docs change i felt this change also can go along with other changes.